packages/vite/src/node/__tests__/plugins/importGlob/fixture-a/index.ts

```diff
@@ -110,3 +110,9 @@ export const cleverCwd2 = import.meta.glob([
   '../fixture-b/*.ts',
   '!**/index.ts',
 ])
+
+export const customBase = import.meta.glob('./**/*.ts', { base: './' })
+
+export const customRootBase = import.meta.glob('./**/*.ts', {
+  base: '/fixture-b',
+})
```

Comment:
```suggestion
})

export const customBaseParent = import.meta.glob('/fixture-b/**/*.ts', {
  base: '/fixture-a',
})
```
I think it'd be nice to have a test case that generates an output that includes `../`.